api/src/main/java/org/openmrs/module/pihcore/config/Components.java

```diff
@@ -18,14 +18,19 @@
         public static final String ONCOLOGY = "oncology";  // program
         public static final String NCD = "ncd";  // program
         public static final String HIV = "hiv";  // program. Needs HIVProgramBundle and Haiti HIV MDS package
+        public static final String HIV_PROGRAM = "hivProgram";
+        public static final String HIV_FORMS = "hivForms";
+        public static final String HIV_INTAKE_FORM = "hivIntakeForm";
         public static final String VCT = "vct";
         public static final String MCH = "mch";  // mchForms + mchProgram
         public static final String MCH_FORMS = "mchForms";  // used by ZL (via "mch")
         public static final String MCH_PROGRAM = "mchProgram";  // needs MCHProgramBundle[ZL], used by ZL (via "mch")
         public static final String VACCINATION_FORM = "vaccination";
         public static final String ZIKA = "zika";  // program
         public static final String COVID19 = "covid19";
+        public static final String COVID19_INTAKE_FORM = "covid19IntakeForm";
```

Comment:
Same thing as above, would be good to note that should should only install COVID_INTAKE_FORM *or* COVID, depending on whether you want the full COVID set or just the intake form.

(And yes, it will be great when we have a easier/more flexible way to config on a form-by-form basis :) )